website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -20,7 +20,15 @@ Before you install Appsmith, ensure the following are installed:
 Follow one of the available guidelines below to configure the Kubernetes cluster.
 
 <Tabs groupId="configurations">
```

Comment:
@danciaclara, @sharat87 and @pratapaprasanna - The Business Edition instructions are updated and ready for review.

Reply:
hey I'm not able to find my comments for that page before it was reworked and moved. Will that be lost?

Reply:
I'm a bit worried that the original Configure and Update section for CE got hidden in the tab now. Should we keep this as a separate page itself?

<img width="1123" alt="Screenshot 2022-12-24 at 02 59 40" src="https://user-images.githubusercontent.com/32227135/209407069-8ab9fb50-89f6-40d3-9f5f-cde2f8fff178.png">
<img width="1249" alt="Screenshot 2022-12-24 at 03 00 01" src="https://user-images.githubusercontent.com/32227135/209407075-37054c4c-5026-4cb7-add7-ebde92f838e9.png">

Reply:
I've hidden the sections as the TOC was cluttered and would have confused users.